src/resources/postcss/tribe-common-admin/settings/_modal-nav.pcss

```diff
@@ -15,7 +15,7 @@
 		margin: 0;
 		padding: 0 var(--tec-spacer-7) var(--tec-spacer-5);
 
-		@media (--viewport-full) {
+		@media (min-width: 500px) {
```

Comment:
Just curious why these aren't variables.

Reply:
Was trying to figure out why they weren't being applied in the correct order.
Turns out it is something in our build process that moves media queries. The solve was actually to add max-widths on some of them - but I didn't go back through and switch them back